.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - vendor
```

Comment:
I would not cache the ``vendor`` folder:

- it can create weird issues if the cache gets corrupted, as Composer will still see the packages as being here
- it will invalidate the Travis cache on each build due to the autoloader being regenerated
- it does not bring much benefit, as installing archives from the cache is fast already